patches/server/0842-Added-fire-tick-delay-config-option.patch

```diff
@@ -0,0 +1,56 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jan Trummer <jan.trummer@hotmail.com>
+Date: Tue, 4 Jan 2022 19:16:24 +0100
+Subject: [PATCH] Added fire-tick-delay config option
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index 5a5db15493cd9b83815c36487c2f38cb8ac76f3a..3cabdb76060314bbcbe8d95f685a5ea93bb1d2ac 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -1019,4 +1019,9 @@ public class PaperWorldConfig {
+     private void minBlockLightForMobSpawning() {
+         this.maxBlockLightForMonsterSpawning = getInt("monster-spawn-max-light-level", maxBlockLightForMonsterSpawning);
+     }
++
++    public int fireTickDelay = 30;
++    private void fireTickDelay() {
++        fireTickDelay = getInt("fire-tick-delay", this.fireTickDelay);
```

Comment:
potentially for merger to look into, this should maybe go under game-mechanics rather than the root of the world settings